.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,58 @@
+<!--
+    Hello! Thanks for submitting a pull request to MusicBrainz Server
+    We appreciate your time and interest in helping our project!
+
+    Use this template to help us review your change. Not everything is required,
+    depending on your change. Keep or delete what is relevant for your change.
```

Comment:
"depending on what you're changing. Keep or delete what is relevant for your change" maybe? (to avoid just saying "your change" every time).

Reply:
This is from [Picard’s PR template](https://github.com/metabrainz/picard/blob/master/.github/PULL_REQUEST_TEMPLATE.md), thus I requested a review from @zas

Reply:
Or even "Not everything is required: keep or delete what is relevant for your change."

Reply:
To follow conversation at https://github.com/metabrainz/musicbrainz-server/pull/1345#discussion_r368143043, I rephrased and split the whole paragraph in 
https://github.com/metabrainz/musicbrainz-server/pull/1345/commits/410198fd412dd45e62862f79effbc293d55fc4c7#diff-195a635ad245ded9076330e31134bd80R3-R8 so as to highlight that unneeded parts for the current change should be removed.